### DISCUSSION TITLE
Access List for Custom Locations

### DISCUSSION THREAD

**Discussion by Jorysm** [NginxProxyManager/nginx-proxy-manager#4241] [Q&A]
Hello, everyone. 
We have a proxy host
![NPM](https://github.com/user-attachments/assets/7a27f27d-939c-4250-af9b-4050964aa153)
Help me solve the problem. I need to add
`allow 192.168.1.0/24;
deny all; `
only for custom locations - /_synapse/admin. How can this be implemented?
![NPM_CL](https://github.com/user-attachments/assets/340186c7-6704-471b-84f7-144494d4e8ae)

**Comment by Realmagnum** [comment]
Make this with addition subdomain entry with custom location.

**Comment by gthbusrr** [comment]
I've a different question...

If I want Access List on the root (`/`) but subdomain have another set of access list. Right now, it's not possible.
